features/update/updateutil/UpdateUtil.java

```diff
@@ -0,0 +1,39 @@
+package update.updateutil;
+
+import io.grpc.StatusRuntimeException;
+import io.temporal.client.WorkflowClient;
+import io.temporal.client.WorkflowNotFoundException;
+import io.temporal.client.WorkflowServiceException;
+import io.temporal.sdkfeatures.Run;
+import io.temporal.sdkfeatures.Runner;
+import org.junit.jupiter.api.Assertions;
+
+public class UpdateUtil {
+
+    public static void RequireNoUpdateRejectedEvents(Runner runner, Run run) {
+        try {
+            var history = runner.getWorkflowHistory(run);
+            var event = history.getEventsList().stream().filter(e -> e.hasWorkflowExecutionUpdateRejectedEventAttributes()).findFirst();
+            Assertions.assertFalse(event.isPresent());
+        } catch(Exception e) {
+            Assertions.fail();
+        }
+    }
+
+    public static String CheckServerSupportsUpdate(WorkflowClient client){
```

Comment:
```suggestion
    public static String checkServerSupportsUpdate(WorkflowClient client) {
```

Is your IDE not correcting the spacing?

Reply:
all spacing looks normal in my IDE, i'll admit I didn't even consider looking at the PR for spacing. I'll look at enabling auto formating

Reply:
Enabled spotless with the same setting as the SDK uses